marketplacetests/tests/test_marketplace_in-app_not_you_login_feature.py

```diff
@@ -0,0 +1,71 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from marionette import Wait
+
+from marketplacetests.firefox_accounts.app import FirefoxAccounts
+from marketplacetests.payment.app import Payment
+from marketplacetests.payment.regions.in_app import InAppPayment
+from marketplacetests.marketplace_gaia_test import MarketplaceGaiaTestCase
+from gaiatest.apps.homescreen.app import Homescreen
+from marketplacetests.marketplace.app import Marketplace
+from gaiatest.apps.homescreen.regions.confirm_install import ConfirmInstall
+
+
+class TestMakeInAppPayment(MarketplaceGaiaTestCase):
+
+    APP_NAME = 'Testing In-App-Payments'
+    APP_TITLE = 'In-App-Payments'
+
+    def setUp(self):
```

Comment:
This `setUp` is quite lengthy and is duplicated in here and in `test_marketplace_make_an_in-app_payment.py` I suggest moving it into `MarketplaceGaiaTestCase` and giving it a specific name, like `install_in_app_payments_test_app`, and then calling that from each `setUp`.